### PR TITLE
Add get-pip.py for Python 2.7

### PR DESCRIPTION
**Pull request by pfmoore**

### CODE REVIEW

**Review by pradyunsg — APPROVED** [review]

**Comment by pradyunsg** [comment]
We need a test, that regenerates the script in CI. :)

**Comment by pfmoore** [comment]
Having a test that isn't "pass" would be a start 🙂 

I assume this can be merged now, and then updated when the actual 21.0 release is published? (I can update 2.7 at that point to use 20.3.4 as well, if that's released by then)?

**Comment by pradyunsg** [comment]
This won't need to be updated, as far as I can tell.

I'll let you hit merge on this. ;)

**Pull request merged**
